index.html

```diff
@@ -101,27 +101,25 @@ <h3>Content IDs (CIDs)</h3>
-            <li>Only SHA-256 (0x12) and Blake3 hashes (0x1e).</li>
-            <li>No blocks.</li>
-            <li>This set of options has the added advantage that no varint processing is required.</li>
+            <li>Only modern CIDv1 CIDs are used, not legacy CIDv0.</li>
+            <li>Only the lowercase base32 multibase encoding (the <code>b</code> prefix) is used for human-readable (and subdomain-usable) string encoding.</li>
+            <li>Only the <code>raw</code> binary multicodec (0x55) and <code>dag-cbor</code> multicodec (0x71), with the latter used only for dCBOR42-conformant DAGs.</li>
+            <li>Only SHA-256 (0x12) and Blake3 hash functions (0x1e), and the latter only in certain circumstances.</li>
+            <li>Regardless of size, resources SHOULD NOT be "chunked" into a DAG or merkle tree (as historically done with UnixFS canonicalization in IPFS systems) but rather hashed in their entirety and content-addressed directly.</li>
+            <li>This set of options has the added advantage that all the aforementioned single-byte prefixes require no additional varint processing or byte-fiddling.</li>
           </ul>
           <p>
-            Supporting two hashes isn't ideal, but having one hash type that can stream the verification of large
-            resources is a plus. Because Blake-3 isn't supported by web browsers, it is strongly recommended that
-            CID producers limit themselves to SHA-256 unless the alternative isn't possible. Implementations intended
-            to run in web contexts are likely to either not verify Blake-3 hashes or to have to dynamically load
-            a Blake-3 library, which may cause latency.
+            Supporting two hashes isn't ideal, but having one hash type that can stream large resources (and do incremental verification mid-stream) is a plus. 
+            Because Blake-3 is still far from being supported by web browsers, it is strongly recommended that CID producers limit themselves to SHA-256 if possible.
+            Implementations intending to run in web contexts are likely to either forego Blake-3 verification in-browser, outsource verification to a trusted component, or to have to dynamically load a Blake-3 library in the browser, which may cause latency.
           </p>
           <p>
             Use the following steps to <dfn id="parse-cid-string">parse a CID string</dfn>:
           </p>
           <ol>
             <li>Accept a string <var>CID</var>.</li>
             <li>Remove the first character from <var>CID</var> and store it in <var>prefix</var>.</li>
-            <li>If <var>prefix</var> is not equal to <code>b</code>, throw an error.</li>
+            <li>If <var>prefix</var> is not equal to <code>b</code>, you are dealing with a non-DASLing CID; particularly if the non-DASL CID is unfamiliar in other ways, you may need to throw an error, warn the user, and/or log the exception.</li>
```

Comment:
```suggestion
            <li>If <var>prefix</var> is not equal to <code>b</code>, throw an error.</li>
```

Noooooo. We don't want to reintroduce optionality. If you want to support IPFS, support IPFS.

We can have an "error handling" section that can give options if needed, but we should keep that orthogonal from the algos.